package.json

```diff
@@ -5,24 +5,20 @@
   "devDependencies": {
     "babel-cli": "^6.24.0",
     "babel-preset-es2015": "^6.24.0",
-    "worker-loader": "^0.8.0"
+    "worker-loader": "^0.8.0",
```

Comment:
I don't know if we can call these dev dependencies, since they are needed at build time.